<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stepButton">
          <property name="text">
           <string>step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Speed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="speedSlider">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>900</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Grid size:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>rows</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rowsSize">
                <property name="singleStep">
                 <number>0</number>
                </property>
                <property name="value">
                 <number>64</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>cols:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="colsSize">
                <property name="value">
                 <number>64</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pattern:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="patternComboBox">
          <item>
           <property name="text">
            <string>Empty</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Oscillator</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Glider</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Beacon</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyPatternButton">
          <property name="text">
           <string>apply pattern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="gameLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
